config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-ec2.yaml

```diff
@@ -204,7 +204,7 @@ periodics:
       - name: CONTROL_PLANE_COUNT
         value: "3"
       - name: CONTROL_PLANE_SIZE
-        value: "c5.18xlarge"
+        value: "c5.4xlarge"
```

Comment:
This is a big decrease. Are you sure if this is OK?

Reply:
/approve
/lgtm

/hold

please remove hold if you are ok with this change @mengqiy @hakuna-matatah

Reply:
Yes, it used to be 18X large until recently. It should be fine IMO.

Reply:
ok cancelling hold

/hold cancel